org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractCommunicationMalcolmTest.java

```diff
@@ -140,7 +140,7 @@ public void testAbortIdleRunnableDevice() throws Throwable {
 	public void testAbortIdlePausableDevice() throws Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "testAbortIdleRunnableDevice" on line 128. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)